deploy/kuberhealthy-prometheus-operator.yaml

```diff
@@ -471,6 +471,8 @@ spec:
           value: "4"
         - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
           value: "true"
+        - name: CHECK_SERVICE_ACCOUNT
```

Comment:
These files under deploy are auto-generated by github actions based on the contents in the helm directory -- there is no need to touch these as they will get overwritten. Placing them in the `values.yaml` and the `khcheck-deployment.yaml` files should avoid this overwrite.

Reply:
Ahh I have seen the create-flat-files.sh but I never opened it, a nice solution.
I will push a fix that just changes the values.yaml.